### DISCUSSION TITLE
Shall we prepare a new release?

### DISCUSSION THREAD

**Discussion by rubenporras** [eclipse/tm4e#512] [General]
Hi,

I would appreciate it we could prepare a new release as we did for LSP4E, there are some improvements and we did not released in a while.

what do you think?

**Comment by mickaelistria** [comment]
I'd be OK with that.
@sebthom what do you think? Is there something critical you think should be worked on before releasing; or is the current state sane enough?

**Reply by sebthom** [reply]
I think the current state is releasable.

**Reply by mickaelistria** [reply]
Done.

**Comment by rubenporras** [comment]
thanks
